server/src/common/configOptions.ts

```diff
@@ -1254,22 +1263,13 @@ export class ConfigOptions {
                 execOutput = child_process.execFileSync('python', commandLineArgs, { encoding: 'utf8' });
             }
 
-            // Parse the execOutput. It should be something like "Python 3.x.y".
-            const match = execOutput.match(/[0-9]+.[0-9]+.[0-9]+/);
-            if (!match) {
-                console.info(`Unable to get Python version from interpreter. Received "${execOutput}"`);
-                return undefined;
-            }
+            const versionJson: { major: number; minor: number } = JSON.parse(execOutput);
```

Comment:
I don't see any error handling here. JSON.parse will throw an exception if it receives anything that's not JSON compliant, so you should be ready to handle that.

Reply:
This whole function is a try-catch block, so handles those cases.